apis/Google.Cloud.Container.V1/docs/history.md

```diff
@@ -0,0 +1,11 @@
+# Version history
+
+# Version 1.1.0, released 2019-12-09
+
+- Some retry settings are now obsolete, and will be removed in the next major version
```

Comment:
These don't have link to the commits, just flagging in case it is relevant.

Reply:
Nope, that's fine. (I'll keep the commit links in some cases, but I don't think they always need to be there.)